src/reducers/books.js

```diff
@@ -0,0 +1,15 @@
+import INITIAL_BOOKS from '../utils/bookObject';
+
+
+const bookList = (state = INITIAL_BOOKS, action) => {
```

Comment:
[Optional]

Since `createStore` has a `preloadedState` argument, I would recommend we use the `INITIAL_BOOKS` in the `createStore` argument.


See https://redux.js.org/api/createstore/